lib/domain.js

```diff
@@ -48,13 +48,61 @@ Object.defineProperty(process, 'domain', {
   }
 });
 
+let asyncHook = null;
```

Comment:
Just call `createHook` immediately, it is simpler. We do that for the the `inspector` integration and soon `trace_events` integration too.

Reply:
Good point!